mne/channels/channels.py

```diff
@@ -688,6 +692,11 @@ def add_channels(self, add_list, force_update_info=False):
         -----
         If ``self`` is a Raw instance that has been preloaded into a
         :obj:`numpy.memmap` instance, the memmap will be resized.
+
+        This function expects an MNE object to be appended (e.g. mne.Raw,
+        mne.Epochs, mne.Evoked). If you simply want to add a channel based on
```

Comment:
should have the links:
```suggestion
        This function expects an MNE object to be appended (e.g. :class:`~mne.Raw`,
        :class:`~mne.Epochs`, :class:`~mne.Evoked`). If you simply want to add a channel based on
```